docs/examples/utils/BufferGeometryUtils.html

```diff
@@ -45,6 +45,34 @@ <h3>[method:BufferAttribute mergeBufferAttributes]( [param:Array attributes] )</
 
     </p>
 
+    <h3>[method:InterleavedBufferAttribute interleaveAttributes]( [param:Array attributes] )</h3>
+    <p>
+    attributes -- Array of [page:BufferAttribute BufferAttribute] instances.<br /><br />
+
+    Interleaves a set of attributes and returns a new array of corresponding attributes that share
+    a single InterleavedBuffer instance. All attributes must have compatible types. If merge does not
+    succeed, the method returns null.
+
+    </p>
+
+    <h3>[method:Number estimateBytesUsed]( [param:BufferGeometry geometry] )</h3>
+    <p>
+    geometry -- Instance of [page:BufferGeometry BufferGeometry] to estimate the memory use of.<br /><br />
+
+    Returns the amount of bytes used by all attributes to represent the geometry.
+
+    </p>
+
+    <h3>[method:BufferGeometry mergeVertices]( [param:BufferGeometry geometry], [param:Number tolerance] )</h3>
+    <p>
+    geometry -- Instance of [page:BufferGeometry BufferGeometry] to merge the vertices of.<br />
+    tolerance -- The maximum allowable difference between vertex attributes to merge. Defaults to 1e-4.<br /><br />
+
+    Returns a new [page:BufferGeometry BufferGeometry] with vertices closer than the provided tolerance merged.
+    All vertex attributes are accounted for.
```

Comment:
Let's explain what "accounted for" means — arguably averaging different vertex colors could be considered "accounted for" too. perhaps, `Only vertices for which all vertex attributes are similar (within tolerance) will be merged`.

Reply:
Fixed. Should be more clear, now.